src/handbook/product/verticals.md

```diff
@@ -1,17 +1,81 @@
 ---
-navTitle: Market Verticals
+navTitle: Product Categories
 ---
 
-# Product Verticals
+# Product Categories
 
-## Background
-Node-RED is best used for connecting _things_ together, and doing logic or manipulation of the data in that connection. Those things can be wide ranging from Web APIs using HTTP to physical hardware on serial connections.
-FlowForge competes in specific market verticals which allow us to optimize for certain use-cases.
+Product verticals refer to specific, identifiable categories within an industry or market that have distinct characteristics and needs. By understanding the verticals, we can align our [product strategy](./strategy.md) to meet those specific needs. For this overview, we will focus on three distinct product verticals: Industrial IoT Platforms, Low-Code Tools, and Data Integration Tools.
```

Comment:
I'd be understanding if we added a statement "In this order" at the end. We need to capture one market first IMO. While we can be serving many, product focus might well be just one for the time being.

Reply:
Not adding "In this order" is fine too, but just sync'ing my thoughts with yours :)

Reply:
What do you think of the term "Low-Code IIoT Platform"?

Reply:
That is indeed what we should be IMO